.github/workflows/test_lint.yml

```diff
@@ -7,32 +7,46 @@ jobs:
     defaults:
       run:
         working-directory: ./app_feup
-    steps: 
+    steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
         with:
           flutter-version: '2.0.1'
+
+      - name: Cache pub dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.FLUTTER_HOME }}/.pub-cache
+          key: ${{ runner.os }}-pub-${{ github.ref }}-${{ hashFiles('**/pubspec.lock') }}
+          restore-keys: ${{ runner.os }}-pub-${{ github.ref }}-
+
```

Comment:
Why are caching the dependencies on this now? Is it just to improve performance?

Reply:
I believe so, I think @reeckset was responsible for this change.